### ISSUE TITLE
PPA doesn't work for Ubuntu 20.04 / focal

### ISSUE THREAD

**Issue by alexislefebvre** [flozz/cover-thumbnailer#7]
On Ubuntu 20.04, I try to use the PPA:

```shell
$ sudo add-apt-repository ppa:flozz/flozz
 This PPA contain ubuntu package of my softwares...
 More info: https://launchpad.net/~flozz/+archive/ubuntu/flozz
Press [ENTER] to continue or Ctrl-c to cancel adding it.

Hit:1 http://archive.canonical.com/ubuntu focal InRelease
Ign:2 http://ppa.launchpad.net/flozz/flozz/ubuntu focal InRelease
Hit:3 http://ppa.launchpad.net/nrbrtx/python2-stuff/ubuntu focal InRelease
Err:4 http://ppa.launchpad.net/flozz/flozz/ubuntu focal Release
  404  Not Found [IP: 2001:67c:[…] 80]
Get:5 http://ubuntu.mirrors.ovh.net/ubuntu focal InRelease [265 kB]
Get:6 http://ubuntu.mirrors.ovh.net/ubuntu focal-updates InRelease [107 kB]
Get:7 http://ubuntu.mirrors.ovh.net/ubuntu focal-security InRelease [107 kB]
Reading package lists... Done 
E: The repository 'http://ppa.launchpad.net/flozz/flozz/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

```

So the `Release` file is missing for `focal`.

**Comment by flozz** [comment]
I do not copied the package for ubuntu 20.04... But seeing PR #5, some dependencies will be missing so we cannot simply copy the package... maybe C-T should be ported to Python 3...

**Comment by flozz** [comment]
I fixed Cover Thumbnailer for Ubuntu 20.04. (CT v0.10.0).

I will not Update the PPA anymore, I have no time to handle this, sorry. :)
